src/vsr/message_header.zig

```diff
@@ -617,6 +622,7 @@ pub const Header = extern struct {
                     // the check requires the StateMachine type.
                 },
             }
+            if (!stdx.zeroed(&self.previous_request_latency_padding)) return "reserved != 0";
```

Comment:
```suggestion
            if (!stdx.zeroed(&self.previous_request_latency_padding)) return "padding != 0";
```

feels useful to distinguish the two when we inevitably mess up versioning one day!